dev/script/start.sh

```diff
@@ -84,7 +88,21 @@ checkRequirements() {
 }
 
 startDockerContainers() {
-  docker-compose up -d
+  if [[ $USE_TEST == true ]]; then
+    docker-compose down
+    EXISTING_TUNNELS=$(ps -ef | grep ssh | grep 9200 | grep -v grep || true)
+    if [[ -n $EXISTING_TUNNELS ]]; then
+      echo "RE-USING EXISTING TUNNEL TO TEST ELASTICSEARCH (on port 9200)"
+    else
+      TUNNEL_OPTS="-o ExitOnForwardFailure=yes -o ServerAliveInterval=10 -o ServerAliveCountMax=2"
+      SSH_COMMAND=$(ssm ssh --profile media-service -t elasticsearch-data,grid-elasticsearch,TEST --newest --raw)
```

Comment:
not blocking, but it would be great to figure out a way to clean up the tunnel when stopping, which would stop accidentally connecting to the cluster (since the tunnel is on the default ES port)

Reply:
ahh in case we switch back to running in normal local mode?

Reply:
yep, or anything else which connects to 9200

Reply:
done!